.github/workflows/build_and_test.yml

```
@@ -0,0 +1,65 @@
+name: Build and Test
+on: [push]
+jobs:
+  tests:
+    runs-on: ubuntu-latest
```

In order to use our beefy runner, you'll need to do this.
```suggestion
    runs-on: [self-hosted, linux]
```

---

Demian says he rebuilt the worker off of ubuntu20 so we might be able to use it again. Let's keep it out of this PR for now though.